pkgs/tools/networking/xh/default.nix

```diff
@@ -38,6 +38,11 @@ rustPlatform.buildRustPackage rec {
       --fish completions/xh.fish \
       --zsh completions/_xh
 
+    installManPage doc/xh.1
+    ln -s ./xh.1.gz $out/share/man/man1/xhs.1
```

Comment:
```suggestion
```
nix automatically compresses the manpage later.

Reply:
I've made the symlink absolute, which makes the manpage compression hook correct it after it's compressed the manpages themselves.

Reply:
This is symlinking the compressed file to xhs